crypto/asn1/a_int.c

```diff
@@ -82,7 +82,7 @@ static void twos_complement(unsigned char *dst, const unsigned char *src,
     dst += len;
     src += len;
     /* two's complement value: ~value + 1 */
-    for (; len > 0; len--) {
+    while (len--) {
```

Comment:
Chosen != 0 as expression of brutal explicitness. I mean what is unsigned being larger than 0 after all :-)